### PR TITLE
space default stream sort

### PR DESCRIPTION
**Pull request by gevorgmansuryan**
https://github.com/humhub/humhub-internal/issues/377

### CODE REVIEW

**Comment by luke-** [comment]
@gevorgmansuryan Thanks!

@yurabakhtin Can you please review?

**Review by yurabakhtin — CHANGES_REQUESTED** [review]

File: protected/humhub/modules/admin/models/forms/SpaceSettingsForm.php [hunk 1]
```diff
@@ -194,4 +205,12 @@ private function updateDefaultSpaces()
         }
     }
 
+    public static function defaultStreamSortOptions(): array
+    {
+        return [
+            Stream::SORT_CREATED_AT => Yii::t('AdminModule.space', 'Creation time'),
+            Stream::SORT_UPDATED_AT => Yii::t('AdminModule.space', 'Last update'),
```
[the remark this answers is not in the document]
Comment (luke-) [reply protected/humhub/modules/admin/models/forms/SpaceSettingsForm.php:212]: @yurabakhtin Good point! No better use the existing message category here.

@gevorgmansuryan Can you please adapt?

[the remark this answers is not in the document]
Comment (yurabakhtin) [reply protected/humhub/modules/admin/models/forms/SpaceSettingsForm.php:212]: @gevorgmansuryan Could you please change the translation category from `AdminModule.space` to `ContentModule.base` for those 2 strings? It makes a work of translators easier :)

[the remark this answers is not in the document]
Comment (gevorgmansuryan) [reply protected/humhub/modules/admin/models/forms/SpaceSettingsForm.php:212]: @yurabakhtin done. sorry just missed this point

**Review by yurabakhtin — APPROVED** [review]

**Pull request merged**